.changeset/plenty-eyes-brush.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/create-app': patch
+---
+
+Add permissions to create-app's PluginEnvironment
+
+`CatalogEnvironment` now has a `permissions` field. This means that the environment parameter passed to `CatalogBuilder.create` in your Backstage backend needs to contain a `permissions` of type `ServerPermissionClient`. See the example backend's [`PluginEnvironment`](https://github.com/backstage/backstage/blob/ffdb98aa2973366d48ff1774a7f892bc0c926e7e/packages/backend/src/types.ts#L29) and how it's initialized [here])(https://github.com/backstage/backstage/blob/ffdb98aa2973366d48ff1774a7f892bc0c926e7e/packages/backend/src/index.ts#L68-L71).
```

Comment:
Let's put the changes that need to be made straight in the changeset to make it as simple as possible to stay up to date. Typically it's some form of "To apply these changes to an existing app, add the following to ...", see other entries in https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md

Reply:
Updated the changeset, but I wasn't 100% confident on if I gave the correct example. It seems like to me that integrators don't necessarily have to use `makeCreateEnv`, so I'm not sure if I need to provide additional context in the code snippet.